CoEpi/ui/resources/fonts/Fonts.swift

```diff
@@ -0,0 +1,39 @@
+
+import UIKit
+
+enum FontNames : String {
+    case robotoRegular = "Roboto-Regular"
+    case robotoLight = "Roboto-Light"
+    case robotoBold = "Roboto-Bold"
+}
+
+class Fonts {
+    static var robotoRegular : UIFont = { guard let font = UIFont(name: FontNames.robotoRegular.rawValue , size: 18)
+        else {
+            fatalError(fatalFontErrorMessage(font: .robotoRegular))
```

Comment:
Should we fatal if a font isnt there?

Reply:
I think it's okay, since it's clearly a programmatic error.

At a later point we can default to preferred fonts and so on but this feels like overkill atm.